subiquity/common/filesystem/manipulator.py

```diff
@@ -48,15 +48,14 @@ def create_filesystem(self, volume, spec):
         if spec.get('fstype') is None:
             # prep partitions are always wiped (and never have a filesystem)
             if getattr(volume, 'flag', None) != 'prep':
```

Comment:
I always find this conditional very confusing somehow. It's just "if the partition is not prep" right? ... Actually, why would this function ever be called for a prep partition? Maybe this should just be an assert or something.

Reply:
Leaving alone for this PR